server/src/test/java/org/elasticsearch/repositories/RepositoriesServiceTests.java

```diff
@@ -181,6 +173,26 @@ public void testRegisterRejectsInvalidRepositoryNames() {
         }
     }
 
+    public void testPutRepositoryVerificationFails() {
```

Comment:
Is it possible to also have a test that starts with an existing repository and demostrate a failed update (due to verfication) request does not change the existing repo?

Reply:
yeah, sure

Reply:
added new test `testPutRepositoryVerificationFailsOnExisting`